### PR TITLE
[IMP] Account: Add fapiao to invoice list view

### PR DESCRIPTION
**Pull request by tanghulu0608**
Description of the issue/feature this PR addresses:
Add fapiao to invoice list view

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by alialfie — COMMENTED** [review]
Hi! Could you please explain the need for this?

**Comment by tanghulu0608** [comment]
Hello. In China fapiao is some thing like a special invoice which comes from the government. It is very useful to show odoo invoice number and fapiao number together in the list view.

**Review by alialfie — CHANGES_REQUESTED** [review]
Thanks for the explanation and sorry for the delay! As confirmed with JVAN, we are okay with this change. I left one comment below, and could I also ask you to target 16.0 instead, since this change doesn't break stable and could be useful in the previous versions too.

**Comment by tanghulu0608** [comment]
hello @alialfie 
I have made a new pr target to 16.0, and add the same invisible conditions as in form view.
#182868

**Comment by tanghulu0608** [comment]
#182869 
This PR is for V17, use 'invisible' in tree view.  It can be forwarded to v18

**Comment by tanghulu0608** [comment]
This PR targeting to master also works. I have tested it too.

File: addons/account/views/account_move_views.xml [hunk 1]
```diff
@@ -515,6 +515,7 @@
                             invisible="context.get('default_move_type') not in ('out_invoice', 'out_refund', 'out_receipt', 'in_invoice', 'in_refund','in_receipt')"/>
                     </header>
                     <field name="made_sequence_gap" column_invisible="True"/>
+                    <field name="country_code" column_invisible="True"/>
```
Comment (alialfie) [inline addons/account/views/account_move_views.xml:518]: You don't need to do this anymore starting from v18 :)

**Review by alialfie — CHANGES_REQUESTED** [review]
Thanks for the quick response! So, we will merge this one for now, and do a backport later for the previous versions. I will be closing the other two PRs. I only have one small comment and after that, it should be good to go!

File: addons/account/views/account_move_views.xml (hunk 1 above)
Comment (tanghulu0608) [reply addons/account/views/account_move_views.xml]: I remove this code, and it works. It's amazing.

File: addons/l10n_cn/views/account_move_view.xml [hunk 2]
```diff
@@ -1,6 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <data>
+        <record id="view_invoice_tree_inherit_i10n_cn" model="ir.ui.view">
+            <field name="name">view.invoice.tree.inherit.i10n.cn</field>
+            <field name="model">account.move</field>
+            <field name="inherit_id" ref="account.view_invoice_tree"/>
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='ref']" position="after">
+                    <field name="fapiao" optional="hide" invisible="country_code != 'CN' or move_type not in ['out_invoice', 'out_refund', 'in_invoice', 'in_refund']"/>
```
Comment (jco-odoo) [inline addons/l10n_cn/views/account_move_view.xml:10]: Not sure that the invisible makes much sense as the field will be empty anyways in these cases...  @alialfie (in the list view, the best thing we have is still the optional hide, I am afraid)

Comment (alialfie) [reply addons/l10n_cn/views/account_move_view.xml:10]: @tanghulu0608 As Josse mentioned, I was incorrect here to suggest the invisible, my bad. Could you please remove the `invisible` attribute again and we can merge? :) Thanks!

Comment (tanghulu0608) [reply addons/l10n_cn/views/account_move_view.xml]: @alialfie  `invisible` attribute removed.

**Pull request closed without merging**